pkg/node/address.go

```diff
@@ -44,6 +44,9 @@ var (
 	ipv4AllocRange      *cidr.CIDR
 	ipv6AllocRange      *cidr.CIDR
 
+	// k8s Node IP (either InternalIP or ExternalIP or nil; the former is prefered)
```

Comment:
Nit: could you update the copyright year? (Github UI doesn't let me select unchanged lines.)